### ISSUE TITLE
ember-test-helpers-codemod fails with errors for nonexisting acceptance tests

### ISSUE THREAD

**Issue by rajasegar** [ember-cli/ember-cli-update-codemods-manifest#124]
`ember-test-helpers-codemod acceptance tests/acceptance` fails in addons
Most addons might not have acceptance tests so there will not be any folder named acceptance tests.

**Comment by kellyselden** [comment]
How should we fix this? Is this something they need to fix? Is their README wrong?

**Comment by rajasegar** [comment]
I don't think we can fix this. One thing we can think about is removing the second command from the manifest only on the assumption that there won't be any acceptance tests. But that is too not the right solution. And there is nothing wrong with their README also.
